src/universal/modules/userDashboard/containers/UserSettings/UserSettingsContainer.js

```diff
@@ -76,7 +88,56 @@ export default class UserSettingsContainer extends Component {
       router.push(nextPage);
     }
     untouch('preferredName');
-  };
+  }
+
+  uploadPicture(pictureFile) {
+    return cashay.mutate('createUserPicturePutUrl', {
+      variables: {
+        userFilename: pictureFile.name
+      }
+    })
+    .then(({data, error}) => {
+      if (error) {
+        throw new Error(error._error); // eslint-disable-line no-underscore-dangle
+      }
+      return data.createUserPicturePutUrl.picturePutUrl;
+    })
+    .then(picturePutUrl => {
+      return fetch(picturePutUrl, {
```

Comment:
You're tellin me that jerk Alice can call fetch & supply `team123`as the team id & replace Parabol's team avatar with a picture full of weiners? Bob ain't gonna be happy.